packages/devtools/lib/src/logging/logging_controller.dart

```diff
@@ -33,6 +33,10 @@ typedef OnLogCountStatusChanged = void Function(String status);
 
 typedef OnShowDetails = void Function({String text, InspectorTree tree});
 
+// TODO(jacobr): remove this typedef if we reimplement package:ansi_up
```

Comment:
The typedef seems unnecessary as is - the function type isn't very long and a typedef that repeats the field name isn't adding information.

I'd do


```dart
final String Function(String) _logMessageToHtml;
```

https://dart.dev/guides/language/effective-dart/design#prefer-inline-function-types-over-typedefs

Reply:
Done. Added this when I thought I had to use the typedef a bunch more places.